src/components/tabs/demoStretchHeight/index.html

```diff
@@ -0,0 +1,35 @@
+<div ng-controller="DemoStretchHeightCtrl" ng-cloak>
+  <md-block style="height: 460px" layout="column">
+    <p class="md-padding">The content in this <code>&lt;p&gt;</code> tag and the <code>md-tab</code> element below are in a 460 pixel tall container. <code>md-stretch-height</code> will automatically account for sibling elements.</p>
+    <md-tabs md-stretch-height>
```

Comment:
Please add styling similar to the demoDynamicHeight so at minimum the `md-tabs` background is white. This seems to work for me locally:
```scss
md-tabs {
  border: 1px solid #e1e1e1;
  md-tabs-wrapper {
    background: white;
  }
}
h1:first-child {
  margin-top: 0;
}
```

Reply:
@Splaktar I have no idea what bug you're referring to as I can't recreate any issue locally. Styling was not necessary as I could see. Adding styles also seems outside the scope of this PR.